### ISSUE TITLE
[Anon] Bug report: Adeptus Astartes.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-killteam#336]
**File:** Adeptus Astartes.catz

**BattleScribe version:** 2.02.03

**Platform:** Android

**Dropbox:** No

**Description:** The data file fails to load while other factions' could. Waited for over 10 minutes and still loading with intense battery consumption.

**Comment by Mad-Spy** [comment]
Duplicate of #321, not strictly a data issue. It's been raised with the app developer, and I'm waiting on a response.
